src/GovUk.Education.ExploreEducationStatistics.Content.Model/GlossaryEntry.cs

```diff
@@ -8,16 +7,16 @@ public class GlossaryEntry
     {
         public Guid Id { get; set; }
 
-        public string Title { get; set; }
+        public string Title { get; set; } = string.Empty;
 
-        public string Slug { get; set; }
+        public string Slug { get; set; } = string.Empty;
 
-        public string Body { get; set; }
+        public string Body { get; set; } = string.Empty;
 
-        public DateTime Created { get; set; }
+        public DateTime Created { get; set; } = new DateTime();
```

Comment:
Is that a warning without it?

We initialise all our times to `DateTime.UtcNow` and what you have wouldn't be a UTC time if somebody accidently forgets to replace it.

Reply:
I cannot remember 100%, but it seems like I added this to remove a `Created.set is never used` warning - although I'm not sure if I agree with my past self that that was the way to do it.

I'll restore the line to how it was for now.